static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -133,6 +133,10 @@ define([
         setListeners = function () {
             googletag.pubads().addEventListener('slotRenderEnded', raven.wrap(function (event) {
                 rendered = true;
+
+                // record first ad on page rendered
+                _.once((new Image()).src = config.page.beaconUrl + '/count/ad-render.gif');
```

Comment:
Why not use the `beacon module`? It abstracts this away and uses the Beacon API if it is available.